docker/Dockerfile

```diff
@@ -8,6 +8,17 @@ ENV \
     NVIDIA_DRIVER_CAPABILITIES="compute,video,utility" \
     NVIDIA_VISIBLE_DEVICES="all"
 
+# The EXPOSE instruction informs Docker that the container listens on the specified network ports at runtime. 
+# The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between 
+# the person who builds the image and the person who runs the container, about which ports are intended to be published
+EXPOSE 8888/tcp
```

Comment:
We probably do not need the full inline description on what a config does. That is what documentation is for. Rather, a comment should describe why it is put here. "Unmanic uses port 8888 by default"